webapp/javascript/services/base.ts

```diff
@@ -107,23 +117,35 @@ export async function request(
         }
       }
 
```

Comment:
@abeaumont @kolesnikovae
you guys may know this better, is any other case we are missing?
eg server responds with an error message in a different field other than errors | error | message

Reply:
I think we are not sending errors in JSON format in all the cases. Some consistency in the API would be helpful here.